exampleSite/content/faq/faq_2.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "FAQ 2"
+date: 2019-04-01T13:30:00-00:00
+description: "Frequently Asked Questions"
+---
+
+This is an example.
+
+sadfasdfa
```

Comment:
Let's use some better text than date. Perhaps we could use some content from: https://wiki.eclipse.org/IRC_FAQ or https://wiki.eclipse.org/The_Official_Eclipse_FAQs